### ISSUE TITLE
CI failing for macOS 10.13 due to scikit-learn 0.24 update

### ISSUE THREAD

**Issue by joshuacwnewton** [neuropoly/spinalcordtoolbox#3121]
`scikit-learn` [just released `0.24`](https://scikit-learn.org/stable/auto_examples/release_highlights/plot_release_highlights_0_24_0.html), and now the CI build for macOS 10.13 is giving this error:

https://travis-ci.com/github/neuropoly/spinalcordtoolbox/jobs/464176777#L1073

```
Check if scikit-learn is installed..................[FAIL]
dlopen(/Users/travis/build/neuropoly/spinalcordtoolbox/python/envs/venv_sct/lib/python3.6/site-packages/sklearn/__check_build/_check_build.cpython-36m-darwin.so, 2): Symbol not found: ____chkstk_darwin
  Referenced from: /Users/travis/build/neuropoly/spinalcordtoolbox/python/envs/venv_sct/lib/python3.6/site-packages/sklearn/__check_build/../.dylibs/libomp.dylib (which was built for Mac OS X 10.15)
  Expected in: /usr/lib/libSystem.B.dylib
 in /Users/travis/build/neuropoly/spinalcordtoolbox/python/envs/venv_sct/lib/python3.6/site-packages/sklearn/__check_build/../.dylibs/libomp.dylib
___________________________________________________________________________
Contents of /Users/travis/build/neuropoly/spinalcordtoolbox/python/envs/venv_sct/lib/python3.6/site-packages/sklearn/__check_build:
__init__.py               __pycache__               _check_build.cpython-36m-darwin.so
setup.py
___________________________________________________________________________
It seems that scikit-learn has not been built correctly.
If you have installed scikit-learn from source, please do not forget
to build the package before using it: run `python setup.py install` or
`make` in the source directory.
```

As a side note, the error references `libomp`, so it may be similar to #3033

**Comment by joshuacwnewton** [comment]
So, the relevant bit of the error message seems to be "Symbol not found: ____chkstk_darwin". From https://github.com/nodegui/nodegui/issues/391: 

> Looks like prebuilt binary support added on the latest version of nodegui (0.13 and up) has backfired in this case.
>
> The binaries are built on Github actions ci server and that uses mac os v10.15. And It seems its not compatible with macos 10.13

I would imagine this is the same case here. `scikit-learn` prebuilds the `libomp` binary (to support users who don't have `libomp` installed), but because they prebuild using 10.15, this isn't compatible with our macOS 10.13 CI build.

I'm going to open an issue on `scikit-learn` to alert them of this.

**Comment by joshuacwnewton** [comment]
Good news! https://github.com/scikit-learn/scikit-learn/issues/19063#issuecomment-750902780 :smile: 

> Thank you very much for the confirmation. We will include this fix in a 0.24.1 release (soon-ish).

**Comment by joshuacwnewton** [comment]
Good news! https://github.com/scikit-learn/scikit-learn/issues/19063#issuecomment-764666537

> This was fixed with the 0.24.1 release a few days ago.

And, as expected, our 10.13 CRON now passes again: https://travis-ci.com/github/neuropoly/spinalcordtoolbox/jobs/473624523
